master/reference/cni-plugin/configuration.md

```diff
@@ -252,7 +254,7 @@ Calico CNI plugin configuration:
 
 When making use of the `usePodCidr` option, the {{site.prodname}} CNI plugin requires read-only Kubernetes API access to the `Nodes` resource.
 
-### IPAM Manipulation with Kubernetes Annotations
+### Using Kubernetes Annotations
```

Comment:
Sentence case for headings....and annotation not a proper noun === lowercase annotations